components/inboxCard/inboxCard.tsx

```diff
@@ -264,7 +275,7 @@ const InboxCard = ({
         {last_message?.type === MessageType.EMAIL ||
         last_message?.type === MessageType.FROM_CONTACT ? (
           <NameText className='description'>
-            <RenderHtml isSpreadIcon={false} htmlstring={description} />
+            <RenderHtml isSpreadIcon={false} htmlstring={combinedDescription} />
```

Comment:
You can try here like `<p>${description}</p>`
after this you not need function for getCombinedDescription